scripts/parse_to_soot.py

```diff
@@ -39,7 +40,8 @@ def export_csv():
                     result_reverse.append([class_name, "sink", line])
 
             if result:
-                class_method_folder = base_path + "/changed-methods/" + scenario[CLASS_NAME] + "/" + scenario[METHOD]
+                class_method_folder = base_path + "/changed-methods/" + class_name + "/" + method
+                class_method_folder = class_method_folder.replace(" ", "")
```

Comment:
Does this not interfere with the Linux path?

Reply:
@galilasmb I do not think so. The problem in this case was that the directory path was being created using method names. e.g.: metodoExample (int i, String s).
For these cases, I just remove the spaces as this did not allow navigation on the windows.